### PR TITLE
Task - Add Case Status On Case Type

### PR DESCRIPTION
**Pull request by stephanieboyd**

### CODE REVIEW

File: static/statuses/models.py [hunk 1]
```diff
@@ -2,11 +2,19 @@
...
+    status = models.CharField(null=False, blank=False, max_length=50)
+    priority = models.IntegerField(null=False, blank=False)
     is_read_only = models.BooleanField(blank=False, null=True)
+
+
+class CaseStatusOnType(models.Model):
+    class Meta:
+        unique_together = (("type", "status"),)
+
+    type = models.CharField(choices=CaseType.choices, null=False, max_length=35)
```
Comment (Mark-Spiers-kainos) [inline static/statuses/models.py]: I don't like the word type here

Comment (BenAndrew-Kainos) [reply static/statuses/models.py]: case_type better?

Comment (Mark-Spiers-kainos) [reply static/statuses/models.py]: yeah

Comment (stephanieboyd) [reply static/statuses/models.py]: @Mark-Spiers-kainos A different change made, following Eugene's inout

**Pull request merged**